sdk/storage/azure-storage-quickquery/src/main/java/com/azure/storage/quickquery/models/BlobQuickQueryDelimitedSerialization.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.quickquery.models;
+
+/**
+ * Defines the input or output delimited (CSV) serialization for a blob quick query request.
+ */
+public class BlobQuickQueryDelimitedSerialization extends
+    BlobQuickQuerySerialization {
+
+    private char columnSeparator;
+    private char fieldQuote;
+    private char escapeChar;
```

Comment:
Is it possible that for example separator can be multi-character? If yes we might reconsider data type here to avoid future migration/break.

Reply:
Added a comment, we can discuss better options